docs/team/becoming-member.md

```diff
@@ -0,0 +1,62 @@
+* Can be elected as the Software Steering Council representative for JupyterLab.
+* Get a vote in a voting situation.
+* Count towards quorum in a voting situation.
+* Are expected to participate in a majority of votes.
+* Can nominate new members.
+* Should actively participate, either synchronously or asynchronously, in meetings.
+
+Inactive members:
+
+* Were previously an active member.
+* Do not vote.
+* Are not counted towards voting quorum.
+* Can "reactivate" at any time by expressing their change in status publicly.
+
+Council members can freely pass between active and inactive at any time. They *should* publicly state their status change in a pull request that updates the `contributors.yaml` file with their status change.
+
+This means an inactive member can "reactivate" themselves at any time by publicly stating their change in status. This does not require a nomination from another team member.
+
+For example, a member who is going out on a long leave/vacation (>2 weeks) can temporarily change their status to inactive during their absence and immediately reactivate upon return. This isn't required, but this can relieve them from having to watch this repository for any formal votes that happen during their absence.
+
+## Nominating a new member
+
+For someone to become a JupyterLab Council member, they should already be a consistent,
+positive, productive member of the community. Newcomers are encouraged to
+become members after they've shown a sustained interest in
+engaging with the community. Moreover, team members should be interested in
+**continuing their engagement** over a long-ish period of time (at least one year), generally
+putting in more time and effort than non-members. This doesn't have to
+mean contributing code - it can be assisting others in forums/issues, reviewing
+pull requests, participating in team meetings, etc.
+
+Any new members must be nominated and championed by an active JupyterLab Council member.
+This process takes the following steps:
+
+1. The champion should first discuss internally with the JupyterLab Council to
+   ensure that there's general consensus before officially starting
+   the process.
+2. If there is obvious consensus within the JupyterLab Council, then move to the
+   next step. If not, then an internal vote can be taken to protect the privacy
+   of the potential member.
+3. A member of the JupyterLab Council contacts the potential new member and asks
+   if they are interested. Don't forget to run them by the {ref}`membership_guidelines`
+   page to make sure they understand what they're signing up for.
+
+## Membership Maintenance
+
+Every six months, one currently active member should open an issue in the team-compass repo asking all currently active team members to reply if they still consider themselves active. If not (or no response is given by a team member), it will be assumed that they have gone inactive. This will help keep the active team up-to-date.
```

Comment:
```suggestion
Every six months, one currently active member should open an issue in the team-compass repo asking all currently active team members to reply if they still consider themselves active. If not (or no response is given by a team member), it will be assumed that they have gone inactive. This will help keep the active member list up-to-date.
```